Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -52,8 +52,8 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
 
     override var text: String? {
         didSet {
-            // SELtextDidChange is not called when directly setting the text property, so fire it manually.
-            SELtextDidChange(self)
+            applyCompletion()
+            super.text = text
```

Comment:
I don't know much about this, but doesn't `textDidChange()` need to be called anymore?